src/widgets/styles/qstylesheetstyle.cpp

```diff
@@ -3685,8 +3685,7 @@ void QStyleSheetStyle::drawControl(ControlElement ce, const QStyleOption *opt, Q
 
             if (btn->features & QStyleOptionButton::HasMenu) {
                 QRenderRule subRule = renderRule(w, opt, PseudoElement_PushButtonMenuIndicator);
-                QRect ir = positionRect(w, rule, subRule, PseudoElement_PushButtonMenuIndicator,
-                                        baseStyle()->subElementRect(SE_PushButtonBevel, btn, w), opt->direction);
+                QRect ir = positionRect(w, rule, subRule, PseudoElement_PushButtonMenuIndicator, opt->rect, opt->direction);
```

Comment:
This line and the `SE_PushButtonContents` change below were a revert of a fix they put in Qt 6.2 that broke button text positions... basically everywhere as far as I can tell